composer.json

```diff
@@ -23,5 +23,8 @@
         "mozilla/wp-sw-manager": "dev-master",
         "marco-c/wp_serve_file": "dev-master"
     },
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "autoload": {
+        "classmap": [ "vendor/mozilla/wp-sw-manager/" ]
```

Comment:
In case you decide for one of e psr standards you need to name it here instead of "classmap"